### DISCUSSION TITLE
What features are nice to haves?

### DISCUSSION THREAD

**Discussion by NathanTarbert** [boxyhq/saas-starter-kit#172] [Ideas]
We would love to hear your feedback about features you think would be valuable. Please list them here:

**Comment by kubeworkz** [comment]
It would be nice to add an admin account to see all accounts. Have an admin dashboard. I think that would be a logical place to put billing and subscription views to get a snapshot of your revenues as a Saas owner.

**Reply by NathanTarbert** [reply]
@kubeworkz, there is a dashboard but it's pretty boilerplate

**Reply by devkiran** [reply]
@kubeworkz That's a good idea. We don't have the dashboard for the SaaS owner yet. An alternative solution would be connect your Postgres database to tools like Budibase or Appsmith and build your own dashboard based on the data in your database.

**Comment by kubeworkz** [comment]
Thanks for the reply. I think I'll give this a go to try and add an admin dashboard to the starter kit.

**Comment by arimanyus** [comment]
Single tenant would be great! I see the current implementation is designed for users who are part of a team but a lot of B2C SaaS' are single-tenant. So the ability to switch between single and multi-tenant would be great!

**Comment by thehashrocket** [comment]
User Roles, or the ability to add them. I would like to be able to create roles such as:

Management, Sales, Account Reps, Etc. And different roles would have access to different features and reporting.

**Comment by austintackaberry** [comment]
This feature request is to add support for JIT auto-provisioning.

That is, when a user first signs up, they enter their email address. Then you can extract the domain to route them to the correct SAML SSO login based on the domain which matches the tenant. And when they come back from the SAML SSO login, you should be able to automatically add them to the Team that has team.id equal to their domain.

**Comment by austintackaberry** [comment]
Would also be great to have license management. So that admins on a team can manage licenses. Bonus points if it can be hooked up to stripe.

**Comment by austinsonger** [comment]
Licensing for a on-premise applications would be nice.
